StyleCop.Analyzers/StyleCop.Analyzers.Test/SpacingRules/SA1028UnitTests.cs

```diff
@@ -216,6 +216,24 @@ public class SA1028UnitTests : CodeFixVerifier
             await this.VerifyCSharpFixAsync(testCode, fixedCode, cancellationToken: CancellationToken.None).ConfigureAwait(false);
         }
 
+        /// <summary>
+        /// This is a regression test for DotNetAnalyzers/StyleCopAnalyzers#1445 "SA1028 falsely reports when
+        /// <c>[assembly: InternalsVisibleTo("...")]</c> is used at the end of file":
+        /// https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/1445
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Fact]
+        public async Task TestWhitespaceBeforeClosingBraceAsync()
+        {
+            string testCode = new StringBuilder()
+                .AppendLine("class ClassName")
+                .AppendLine("{")
+                .Append(" }")
+                .ToString();
```

Comment:
I never looked at the complete file, just at your changes :blush:
I don't like the style, but consistency is the more important factor.